cluster-management/setup/mounting-storage/index.md

```diff
@@ -108,6 +108,26 @@ coreos:
 
 Note the declaration of `ConditionPathExists=!/var/lib/docker.btrfs`. Without this line, systemd would reformat the btrfs filesystem every time the machine starts.
 
+## Mounting NFS shares
+
+This example will show how you can mount NFS share. First of all we will enable rpc-statd for NFS host monitoring. Then we will mount `/var/www` share into local `/var/www` path. Please pay attention on the unit name `var-www.mount` it should correspend to mount path `/var/www`.
+
+```yaml
+#cloud-config
+coreos:
+  units:
+    - name: rpc-statd.service
+      command: start
+      enable: true
+    - name: var-www.mount
+      command: start
+      content: |
+        [Mount]
+        What=nfs.example.com:/var/www
+        Where=/var/www
+        Type=nfs
+```
+
```

Comment:
Since you had issues with stuff depending on your example mount, should we add a note about that here?

"If you have units that dep. on this, you might need to X and Y"